### PR TITLE
Add device integration page

### LINKED ISSUE #1118: Add Device Integration page
### Description

We need to create a page that documents how hardware vendors can pre-load our device agent on to their hardware platform.

### Epic/Story

_No response_

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Created the `/certify-hardware/` page

## Related Issue(s)

closes #1118 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-FlowForge** [comment]
@iskerrett @MarianRaphael should I add a link to this page under the product menu? or does it belong somewhere else?

File: src/partners/certify-hardware.njk [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                </div>                
+                <div class="sm:row-span-3 col-span-3 my-auto pt-6 sm:pt-0">Users can easily upload their device configuration either through a custom user interface (e.g., file upload or input mask) or via the built-in Device Agent UI.</div>
+            </div>
+        </div>        
+        {% include "components/divider-grey-wires-right.njk" %}
+    </div>
+    <div class="container max-w-screen-md m-auto flex flex-col gap-4">
+        <h3 class="m-auto text-center w-full">Certification</h3>
+        <div class="flex flex-col md:flex-row md:gap-10 gap-6 md:pb-6 md:px-10">
+            <p class="md:w-[65%] w-full">Lorem ipsum dolor sit amet consectetur. A turpis amet ullamcorper volutpat tellus ut sagittis sit.</p>
```
Comment (Yndira-FlowForge) [inline src/partners/certify-hardware.njk:57]: @MarianRaphael this is the placeholder text we talked about

**Comment by Yndira-FlowForge** [comment]
@iskerrett @MarianRaphael I just added the FlowFuse certified device logo.

Once we've replaced the placeholder text in the "certification" section, it will be fully ready for review.

These are the pages that have been updated:
https://deploy-preview-1248--flowforge-website.netlify.app/partners/
https://deploy-preview-1248--flowforge-website.netlify.app/partners/certify-hardware/

File: src/partners/certify-hardware.njk (hunk 1 above)
Comment (MarianRaphael) [reply src/partners/certify-hardware.njk:57]:
```suggestion
            <p class="md:w-[65%] w-full">After successfully navigating the technical qualification, take the next step by booking a meeting with our team. Together, we'll work towards ensuring your device is fully compatible with FlowFuse and Node-RED</p>
```

**Review by MarianRaphael — APPROVED** [review]
Looks excellent, I suggested a change for the placeholder text.

File: src/partners/certify-hardware.njk [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        {% include "components/divider-grey-wires-right.njk" %}
+    </div>
+    <div class="container max-w-screen-md m-auto flex flex-col gap-4">
+        <h3 class="m-auto text-center w-full">Certification</h3>
+        <div class="flex flex-col gap-6 md:pb-6 md:px-10">
+            <p class="w-full">After successfully navigating the technical qualification, take the next step by booking a meeting with our team. Together, we'll work towards ensuring your device is fully compatible with FlowFuse and Node-RED.</p>
+            <ul class="flex flex-row justify-center w-full items-center">
+                <li class="pb-6 hover:transition-opacity hover:opacity-75 transition ease-in duration-300">
+                    <a href="/book-demo/" class="inline-block ff-btn ff-btn--primary uppercase w-[218px] m-auto text-center">
+                        <span>BOOK A MEETING</span>
```
[the remark this answers is not in the document]
Comment (MarianRaphael) [reply src/partners/certify-hardware.njk:61]: Yes

[the remark this answers is not in the document]
Comment (Yndira-FlowForge) [reply src/partners/certify-hardware.njk:61]: I just replaced the link, in the future I might change it to be a pop-up

**Review by iskerrett — APPROVED** [review]

**Pull request merged**